sections/featured-collection.liquid

```diff
@@ -188,7 +188,7 @@
       </div>
     {%- endif -%}
     {% if section.settings.image_shape == 'arch' %}
-      {% render 'mask-arch' %}
+      {{ 'mask-arch.svg' | inline_asset_content }}
```

Comment:
This doesn't apply the Arch shape to the product card images anymore.

| Current | Expected |
|--------|--------|
| ![](https://screenshot.click/14-49-6k9o3-uzaw1.png) | ![](https://screenshot.click/14-52-7ttlk-l7yl9.png) |

Reply:
That's weird.  There should be no changes to what's rendering.  I will investigate.

Reply:
Yeah, I tried on the spin instance and it was not adjusting the shape either. Let's see! Thanks for taking a look!